packages/components/src/animate/index.js

```diff
@@ -23,13 +55,15 @@ export function useAnimate( { type, origin = getDefaultOrigin( type ) } ) {
 		);
 	}
 
-	if ( type === 'loading' ) {
-		return classnames( 'components-animate__loading' );
-	}
+	return undefined;
```

Comment:
What's the use of explicitly returning undefined?

Reply:
Our TypeScript configuration requires a return type when the type is like `string | undefined`. I've changed it to `string | void` which is largely synonymous, but allows the function to omit a return statement and implicitly return `undefined`.